packages/expo-network/src/Network.ts

```diff
@@ -36,7 +36,7 @@ export async function getNetworkStateAsync(): Promise<NetworkState> {
 
 // @needsAudit
 /**
- * Gets the device's current IPv4 address. Returns `0.0.0.0` if the IP address could not be retrieved.
+ * Gets the device's current IPv4 address. Returns `0.0.0.0` if the IP address is not retrieved.
```

Comment:
The original wording reads more correctly to me

Reply:
In that case, I'll revert it back.

Reply:
Done ✅